Ask index panic

Just saw this panic when running the server:
```
panic: runtime error: index out of range [0] with length 0

goroutine 53 [running]:
github.com/textileio/filecoin/index/ask.calculateMedian(0xc0003541e0, 0x0)
	/Users/aaron/code/textile/filecoin/index/ask/ask.go:263 +0x3d6
github.com/textileio/filecoin/index/ask.(*AskIndex).updateIndex(0xc000426000, 0x0, 0x0)
	/Users/aaron/code/textile/filecoin/index/ask/ask.go:224 +0x760
github.com/textileio/filecoin/index/ask.(*AskIndex).refreshAsks(0xc000426000, 0x0, 0x0)
	/Users/aaron/code/textile/filecoin/index/ask/ask.go:149 +0x79
github.com/textileio/filecoin/index/ask.(*AskIndex).runBackgroundAskCache(0xc000426000)
	/Users/aaron/code/textile/filecoin/index/ask/ask.go:131 +0x95
created by github.com/textileio/filecoin/index/ask.New
	/Users/aaron/code/textile/filecoin/index/ask/ask.go:87 +0x23d
```

--- a/index/ask/ask.go
+++ b/index/ask/ask.go
@@ -251,6 +251,9 @@ func (ai *AskIndex) Close() error {
 }
 
 func calculateMedian(index map[string]StorageAsk) uint64 {
+	if len(index) == 0 {
+		return 0
+	}
 	prices := make([]uint64, 0, len(index))
 	for _, v := range index {
 		prices = append(prices, v.Price)

